### PR TITLE
Added DinoSoft.Folder Prettifier Version 2.0.0

### PR DESCRIPTION
**Pull request by yogesh-aggarwal**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)? Yes!
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change? Yeah!
- [x] Have you tested your manifest locally with `winget install -m <manifest>`? Working totally fine 😊
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting? Yes!

-----

Kindly merge this request because this is a very useful program that is meant for use as a daily driver!! It helps a lot in managing the folders!!

Thanks for the opportunity!

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/4619)

### CODE REVIEW

File: manifests/DinoSoft/FolderPrettifier/2.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+InstallerType: exe
+Installers:
+  # - Arch: x86
+  #   Url: https://github.com/yogesh-aggarwal/folder-prettifier/releases/download/v2.0.0/Folder.Prettifier.32-bit.exe
+  #   Sha256: 51b505a033eaffdd887b27c897f547c659bbfe1e335f2e538e5691d11634d68e
+  #   Scope: machine
+  #   Language: en-US
+  - Arch: x64
+    Url: https://github.com/yogesh-aggarwal/folder-prettifier/releases/download/v2.0.0/Folder.Prettifier.64-bit.exe
+    Sha256: 775acb8f21cf8114c77ff071cec6e9ff67874b707baccfff104804a9e7e7d4d2
```
Comment (yigitemres) [inline manifests/DinoSoft/FolderPrettifier/2.0.0.yaml:25]:
```suggestion
    Sha256: 775acb8f21cf8114c77ff071cec6e9ff67874b707baccfff104804a9e7e7d4d2
    Switches:
      Silent: /quiet
      SilentWithProgress: /passive    
```

**Review by yigitemres — APPROVED** [review]

**Comment by yigitemres** [comment]
@yogesh-aggarwal Don't get me wrong but this pr is somehow suspicious. MSI installer is wrapped, creator of the software is you and `Publisher` name on the manifest kinda misleading.

**Comment by yogesh-aggarwal** [comment]
I have created it using the Advanced Installer, is there any problem in using this?

**Pull request closed without merging**